dirvish.el

```diff
@@ -702,8 +703,8 @@ buffer, it defaults to filename under the cursor when it is nil."
             ((and (not remote) (not (equal flags dired-actual-switches)))
              (dired-sort-other flags))
             ((eq dired-auto-revert-buffer t) (revert-buffer))
-	    ((functionp dired-auto-revert-buffer)
-	     (when (funcall dired-auto-revert-buffer dir) (revert-buffer))))
+            ((functionp dired-auto-revert-buffer)
+             (when (funcall dired-auto-revert-buffer dir) (revert-buffer))))
```

Comment:
This whitespace change is perhaps a little overzealous. I can remove if preferred.